### PR TITLE
fix:在imageDecoder.tracks未完全填充时使用其参数，导致获取的数据有问题

### PR DESCRIPTION
**Pull request by x007xyz**

### CODE REVIEW

**Comment by hughfenghen** [comment]
能否以单测、或 codesandbox 项目复现问题？

新增代码 `await imageDecoder.tracks` 应该没有作用，因为 `imageDecoder.tracks` 不是一个 Promise

**Comment by x007xyz** [comment]
应该是`await imageDecoder.tracks.ready;`，已重新提交

**Comment by x007xyz** [comment]
测试结果截图：
image
测试图片：
氛围_礼花
测试Demo地址：
[Demo](https://codesandbox.io/p/sandbox/webav-demo-83y2gx?file=%2Fsrc%2Findex.ts%3A104%2C24&layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522clv4yhwbu00063b6nolvesyil%2522%252C%2522sizes%2522%253A%255B100%252C0%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522clv4yhwbu00023b6n7y6a47sf%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522clv4yhwbu00033b6nvhyhxubb%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522clv4yhwbu00053b6nnxujt1hj%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%252C%2522sizes%2522%253A%255B50%252C50%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522clv4yhwbu00023b6n7y6a47sf%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clv4yhwbt00013b6nxhxdrski%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522filepath%2522%253A%2522%252Fsrc%252Findex.ts%2522%252C%2522state%2522%253A%2522IDLE%2522%252C%2522initialSelections%2522%253A%255B%257B%2522startLineNumber%2522%253A104%252C%2522startColumn%2522%253A24%252C%2522endLineNumber%2522%253A104%252C%2522endColumn%2522%253A24%257D%255D%257D%255D%252C%2522id%2522%253A%2522clv4yhwbu00023b6n7y6a47sf%2522%252C%2522activeTabId%2522%253A%2522clv4yhwbt00013b6nxhxdrski%2522%257D%252C%2522clv4yhwbu00053b6nnxujt1hj%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clv4yhwbu00043b6n27n6bgh4%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522UNASSIGNED_PORT%2522%252C%2522port%2522%253A0%252C%2522path%2522%253A%2522%252F%2522%257D%255D%252C%2522id%2522%253A%2522clv4yhwbu00053b6nnxujt1hj%2522%252C%2522activeTabId%2522%253A%2522clv4yhwbu00043b6n27n6bgh4%2522%257D%252C%2522clv4yhwbu00033b6nvhyhxubb%2522%253A%257B%2522tabs%2522%253A%255B%255D%252C%2522id%2522%253A%2522clv4yhwbu00033b6nvhyhxubb%2522%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Afalse%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D)
MDN文档说明：
[地址](https://developer.mozilla.org/en-US/docs/Web/API/ImageTrackList)
image

File: packages/av-cliper/src/av-utils.ts [hunk 1]
```diff
@@ -93,6 +93,8 @@ export async function decodeImg(
   const imageDecoder = new ImageDecoder(init);
   await imageDecoder.completed;
 
+  await imageDecoder.tracks.ready;
```
Comment (hughfenghen) [inline packages/av-cliper/src/av-utils.ts:96]: 建议改成
```js
await Promise.all([imageDecoder.completed, imageDecoder.tracks.ready])
```

**Comment by hughfenghen** [comment]
@x007xyz 感谢你的贡献。
查看 Review 记录，略微调整一下代码就可以 Merge 了

**Comment by hughfenghen** [comment]
已发布  @webav/av-cliper@0.4.5

**Pull request merged**